eslint.config.js

```diff
@@ -18,6 +18,9 @@ export default antfu(
         'command/command': 'off',
         'antfu/top-level-function': 'off',
         'style/max-statements-per-line': 'off',
+        'style/no-multi-spaces': 'off',
+        'style/comma-spacing': 'off',
+        'antfu/consistent-list-newline': 'off',
```

Comment:
We shouldn't disable them globally to support this optional command.

Instead, we could document and tell users to disable these rules with inline comments.

Reply:
This was the only way I was able to get past the linter hooks, I tried disabling the specific markdown file but it didn't seem to work, not sure why exactly. I definitely agree it should be mentioned in the docs, I will make an update on this.

Reply:
I added the docs and edited the ESLint file removing the global rules. I ended up having to ignore the markdown file entirely because even if you disable the right rules, the example I provided at the end to demonstrate the ESlint ignore usage will complain about unused eslint-disable directive, which I have not idea how to suppress...

Reply:
Just found out about eslint-skip 😂. It should work now without ignoring the file or disabling global eslint rules.